frontend/src/app/components/FeaturedExperiment/stories.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import moment from 'moment';
+
+import FeaturedExperiment from './index';
+import FeaturedStatus from './FeaturedStatus';
+import FeaturedButton from './FeaturedButton';
+import LayoutWrapper from '../LayoutWrapper';
+
+const experiment = {
+  title: 'Voice Fill',
+  description: 'This is a different experiment This Source Code Form is subject to the terms of the Mozilla Public This Source Code Form is subject to the terms of the Mozilla Public This Source Code Form is subject to the terms of the Mozilla Public.',
+  subtitle: 'A subtitle',
+  slug: 'voice-fill',
+  enabled: true,
+  platforms: ['addon'],
+  survey_url: 'https://example.com',
+  created: '2010-06-21T12:12:12Z',
+  modified: '2010-06-21T12:12:12Z',
+  video_url: 'https://www.youtube.com/embed/wjvM18UWMmM'
+};
+
+const baseProps = {
```

Comment:
Clicking on the install button in the first story, I get an `installAddon is not a function` error. Should add `installAddon: action('installAddon')` here to catch that - or even better, something that might simulate an install with a delay